src/wwwroot/bower.json

```diff
@@ -1,24 +1,26 @@
 {
```

Comment:
This file has been changed because I installed textAngular and foundation-icon-fonts to bower_components